app/models/supporting_evidence_attachment.rb

```diff
@@ -19,4 +19,26 @@ def set_file_data!(file, file_password = nil)
   def converted_filename
     JSON.parse(file_data)['converted_filename']
   end
+
+  def original_filename
+    JSON.parse(file_data)['filename']
+  end
+
+  # Obfuscates the attachment's file name for use in mailers, so we don't email PII
+  # The intent is the file should still be recognizable to the veteran who uploaded it
+  # Follows these rules:
+  # - Only masks filenames longer than 5 characters
+  # - Masks letters and numbers, but preserves special characters
+  # - Includes the file extension
+  def obscured_filename
+    extension = original_filename[/\.\w*$/]
```

Comment:
Note to self: these regexes should live in a constant at the top of the model with a clear name

Reply:
Fixed in https://github.com/department-of-veterans-affairs/vets-api/pull/16359/commits/0d5dbef1bf2e558f19e3b5536a5b8b0378ba265e